charts/jaeger/templates/es-user-config.yaml

```diff
@@ -0,0 +1,62 @@
+{{ if (and (not .Values.userconfig) .Values.provisionDataStore.elasticsearch) }}
```

Comment:
should this file be called "user config"? It looks like a built-in config. Also it's not even a config, but a configmap, so let's call it that.

Reply:
Ok, I will rename it to `es-configmap.yaml `